api/api.proto

```diff
@@ -370,6 +372,10 @@ message VolumeSpecUpdate {
   oneof sharedv4_opt { bool sharedv4 = 24; }
   // QueueDepth defines the desired block device queue depth
   oneof queue_depth_opt { uint32 queue_depth = 25; }
+  // Encrypted is true if this volume will be cryptographically secured.
+  oneof encrypted_opt { bool encrypted = 26; }
```

Comment:
We had a conversation offline, and we will be creating VolumeSpecPolicy as a new message that will house values that need to override the spec. We cannot use VolumeSpecUpdate because these values are the ones that support being updated only.